src/fdk_model_publisher/service/mapper.py

```diff
@@ -72,6 +72,14 @@ def map_model_from_dict(
     model.modelelements = model_element_mapper.extract_model_items(
         partial_model.schema, partial_model.endpoint_description
     )
+    format = FoafDocument()
```

Comment:
Kanskje bedre å kalle denne "document" da hasFormat er range Document. format er en property på document.

Reply:
Grunnen til at det heter FoafDocument og ikke Document er fordi Document er en annen klasse i dcat som er superklassen til FoafDocument. Men dette vil @stigbd kunne forklare deg mer om hvorfor det er slik @oyvindis

Reply:
Jeg tenkte på variabelnavnet, at den kan endres fra "format" til f.eks. "foaf_document", for lesbarheten sin skyld.

Reply:
Det kan vi endre @oyvindis om du vil

Reply:
Eg synes også `foaf_document` er betre. (Det bør i alle tilfeller ikkje vere  `format`.)